PARAM_DIFFERENCES.md

```diff
@@ -0,0 +1,57 @@
+## Parameter Differences
+
+### RDSE
+| HTMCore    |                                                                                                                                                                                                                                                                                            |              | Nupic CPP  |                                                                                                                                                                                                                                                                                                                 |         |
+|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| Attribute  | Description                                                                                                                                                                                                                                                                                | Default      | Attribute  | Description                                                                                                                                                                                                                                                                                                     | Default |
+| size       | Member "size" is the total number of bits in the encoded output SDR\.                                                                                                                                                                                                                      | 0            | n          | Number of bits in the representation \(must be > w\)\. n must be large enough such that there is enough room to select new representations as the range grows\. With w=21 a value of n=400 is typical\. The class enforces n > 6\*w\.                                                                           | 400     |
```

Comment:
nit: if you want, you could enrich the doc (c++ .hpp, py) of the arguments with "(say for size) This used to be `n` in old NuPIC."

Reply:
Done 👍